README.md

```diff
@@ -1,22 +1,19 @@
 # Comments Microservice
```

Comment:
This PR alters the repo README. Please commit again and include an updated README.